app/src/components/dashboard/Callouts.tsx

```diff
@@ -6,6 +6,45 @@ import { cn } from "@/lib/utils"
 
 import ExternalLink from "../ExternalLink"
 
+export function GovCandidateCallout() {
+  return (
+    <ExternalLink
+      href="https://gov.optimism.io/t/season-7-elections-information/9392#p-42386-for-candidates-1"
+      className={cn(
+        "bg-[#D6E4FF] flex flex-col gap-y-6 mt-6 w-full max-w-4xl rounded-lg p-6",
```

Comment:
this is a new bg color that we've not used anywhere else -- let's define it in `globals.css` & `tailwind.config.ts`